<launch>
	<arg name="machine" default="localhost" />
	<arg name="user" default="" />

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>

	<!-- scitos_modules argument; must be a space seperated list of hardware modules to load -->
	<arg name="SCITOS_MODULES" default="Drive Charger EBC Display Head"/>

	<node pkg="scitos_mira" type="scitos_node" name="scitos_node" output="screen">
		<param name="config_file" value="$(find scitos_mira)/resources/SCITOSDriver.xml" type="string"/>
		<param name="scitos_modules" value="$(arg SCITOS_MODULES)" />
	</node>
</launch>
